packages-new/components/accordion/documentation/specifications/specifications-accordion-group.mdx

```diff
@@ -1,6 +1,6 @@
 import {Description} from '@storybook/addon-docs';
-import Specs from '@ovhcloud/ods-core/src/components/accordion-group/docs/spec.md';
-import SpecsAccordionGroupContents from '@ovhcloud/ods-specifications/components/accordion-group/specifications-accordion-group-contents.mdx';
+import Specs from './accordionGroupSpec.md';
```

Comment:
can you keep the previous generic name instead, those file will have to be worked on, so it will be easier if all component have the same